tests/unit/components/notifications/HdNotifications.spec.js

```diff
@@ -43,6 +43,7 @@ describe('HdNotifications', () => {
     wrapper.setMethods({ getScrollHeight: mockedScrollHeight });
     wrapper.setProps({ notifications: NOTIFICATIONS.slice(-1) });
 
+    await wrapper.vm.$nextTick();
```

Comment:
It is super weird. I just run the tests locally and it worked. Have you checked node version? I mean, is it getting the right version from nvm?

Reply:
I'm some versions above, I'm trying to setup nvm on Windows and will update the results as soon as it works.

Reply:
That made the trick. I was two versions above and didn't see that before, thanks for the heads-up.

Reply:
Cool! "Nice" to know that you've had some issues with new versions, we must keep 👀 once it is upgraded 😬